packages/marketplace/contracts/libraries/LibOrder.sol

```diff
@@ -119,32 +121,28 @@ library LibOrder {
     /// @param leftTakeValue The amount the left order taker wants in return.
     /// @param rightMakeValue The amount the right order maker wants to trade.
     /// @param rightTakeValue The amount the right order taker wants in return.
-    /// @return The filled amounts for both the left and right orders.
-    function _fillRight(
+    function _canFillRight(
         uint256 leftMakeValue,
         uint256 leftTakeValue,
         uint256 rightMakeValue,
         uint256 rightTakeValue
-    ) internal pure returns (FillResult memory) {
+    ) internal pure {
         uint256 makerValue = LibMath.safeGetPartialAmountFloor(rightTakeValue, leftMakeValue, leftTakeValue);
         require(makerValue <= rightMakeValue, "fillRight: unable to fill");
-        return FillResult(rightTakeValue, makerValue);
```

Comment:
you didn't have to do any changes in the tests when you actually changed this
```return FillResult(rightTakeValue, makerValue);```
to
```return FillResult(rightTakeValue, rightMakeValue);```
?